crypto/evp/digest.c

```diff
@@ -494,14 +498,27 @@ int EVP_Digest(const void *data, size_t count,
     return ret;
 }
 
+int EVP_MD_CTX_setparams(EVP_MD_CTX *ctx, OSSL_PARAM *params)
```

Comment:
Elsewhere, these have been named `*_set_params`...  I have nothing against `setparams` per se, but we should keep the same name standard everywhere.